mlflow/server/auth/basic_auth.ini

```diff
@@ -0,0 +1,3 @@
+[mlflow]
+default_permission = READ
+database_uri = sqlite:///basic_auth.db
```

Comment:
Can we make the path to this file configurable via an environment variable?

Reply:
added a new env var `MLFLOW_AUTH_CONFIG_PATH`

Reply:
thanks for the update!